python/mlc_chat/compiler/quantization/quantization.py

```diff
@@ -67,4 +67,13 @@ def quantize_weight(self, weight: tvm.runtime.NDArray) -> List[tvm.runtime.NDArr
         storage_dtype="uint32",
         model_dtype="float16",
     ),
+    "q4f16_awq_gemm": AWQQuantize(
+        name="q4f16_awq_gemm",
+        kind="awq",
+        group_size=128,
+        quantize_dtype="int4",
+        storage_dtype="uint32",
+        model_dtype="float16",
+        gemm=True,
+    ),
```

Comment:
Given the default format from AutoAWQ is `awq_gemm`, let's simply rename it to `q4f16_autoawq`, and remove `q4f16_awq`